### PR TITLE
Uml 1161 older lpa - success journey

### PR DESCRIPTION
**Pull request by cooperaj**
# Purpose

Allows the user to request an activation key

Fixes UML-1161

## Approach

Hooked up UI pages to the API responses to complete older LPA journeys. Amended tests. Did lots of refactoring

## Checklist

* [x] I have performed a self-review of my own code
* [x] I have added relevant logging with appropriate levels to my code
* [ ] ~I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant~
* [x] I have added tests to prove my work
* [ ] ~I have added welsh translation tags and updated translation files~
* [ ] ~I have run an accessibility tool on any pages I have made changes to and fixed any issues found~
* [x] The product team have tested these changes

### CODE REVIEW

File: service-api/seeding/seeding_lpa_codes.json [hunk 1]
```diff
@@ -98,17 +98,6 @@
     "generated_date": "2020-06-22",
     "status_details": "Imported"
   },
-  {
```
Comment (GemTay) [inline service-api/seeding/seeding_lpa_codes.json:101]: why was this removed?

Comment (cooperaj) [reply service-api/seeding/seeding_lpa_codes.json:101]: Needed an LPA to be able to add using the older lpa feature, and we add codes for everything (as best as I can tell)

Comment (cooperaj) [reply service-api/seeding/seeding_lpa_codes.json:101]: Needed an LPA to be able to add using the older lpa feature, and we add codes for everything (as best as I can tell)

File: service-front/app/src/Actor/templates/actor/send-activation-key-confirmation.html.twig [hunk 2]
```diff
@@ -21,20 +21,19 @@
                 </div>
                 <div class="govuk-grid-column-two-thirds">
 
-                    <h2 class="govuk-heading-m">{% trans %}What happens next{% endtrans %}</h2>
+                    <h2 class="govuk-heading-l">{% trans %}What happens next{% endtrans %}</h2>
 
                     <p class="govuk-body">{% trans %}We’ll send a letter to the address we have for you. The letter will have your activation key.{% endtrans %}</p>
-                    <p class="govuk-body">{% trans with {'%date%': lpa_date(date)} %}If you do not get the letter by %date%, please contact us.{% endtrans %}</p>
-                    <p class="govuk-body">{% trans %}When you get in touch, quote the OPG reference number(also called LPA reference number).{% endtrans %}</p>
+                    <p class="govuk-body">{% trans with {'%date%': lpa_date(date)} %} If you do not get the letter by %date%, please contact us. When you get in touch, quote the LPA reference number (also called OPG reference number).{% endtrans %}</p>
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/send-activation-key-confirmation.html.twig:27]:
```suggestion
                    <p class="govuk-body">{% trans with {'%date%': lpa_date(date)} %}If you do not get the letter by %date%, please contact us. When you get in touch, quote the LPA reference number (also called OPG reference number).{% endtrans %}</p>
```

File: service-front/app/src/Actor/templates/actor/send-activation-key-confirmation.html.twig [hunk 3]
```diff
@@ -21,20 +21,19 @@
...
+                        <a class="govuk-link" href="https://www.gov.uk/call-charges">{% trans %}Find out about call charges{% endtrans %}</a>
                     </p>
-                    <p class="govuk-body">{% trans %}When you have your activation key, you can use it to add an LPA to your online account. This can then use the LPA online when you need to.{% endtrans %}</p>
+                    <p class="govuk-body">{% trans %}When you have your activation key, you can use it to add an LPA to your online account. You can then use the LPA online when you need to.{% endtrans %}</p>
 
-                    <a href="{{ path('lpa.dashboard') }}"  draggable="false" class="govuk-button">{% trans %}Back to LPAs{% endtrans %}</a>
-                    <a class="govuk-button govuk-button--secondary govuk-!-margin-bottom-1" href="{{ path('logout') }}">
-                        {% trans %}Sign out{% endtrans %}
+                    <a href="{{ path('lpa.add-by-paper') }}"  draggable="false" class="govuk-button">{% trans %}Request another activation key{% endtrans %}</a>
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/send-activation-key-confirmation.html.twig:34]:
```suggestion
                    <a href="{{ path('lpa.add-by-paper') }}" draggable="false" class="govuk-button">{% trans %}Request another activation key{% endtrans %}</a>
```

**Review by GemTay — APPROVED** [review]
